pyproject.toml

```diff
@@ -0,0 +1,44 @@
+[project]
+name = "lamar"
+description = "LaMAR: Benchmarking Localization and Mapping for Augmented Reality"
+version = "1.0"
+authors = [{name = "Microsoft"}]
+readme = "README.md"
+requires-python = ">=3.8"
+license = {file = "LICENSE-CODE"}
+classifiers = [
+    "Programming Language :: Python :: 3",
+    "License :: OSI Approved :: MIT License",
+    "Operating System :: OS Independent",
+]
+dependencies = [
+    "numpy",
+    "scipy",
+    "h5py",
+    "torch>=1.1",
+    "tqdm>=4.36.0",
+    "opencv-python",
+    #"pycolmap @ git+https://github.com/colmap/pycolmap.git@v0.4.0",
```

Comment:
What happen if these lines are uncommented? Just to avoid they were left commented accidentally.

Reply:
My bad, indeed shouldn't have been commented out. Now fixed.